Project-II-Calculator/calculator/src/components/DisplayComponents/CalculatorDisplay.js

```diff
@@ -1,2 +1,53 @@
 import React from 'react';
 import './Display.css';
+import NumberButton from '../ButtonComponents/NumberButton';
+import ActionButton from '../ButtonComponents/ActionButton';
+import ClearButton from '../ButtonComponents/ClearButton';
+import Zero from '../ButtonComponents/Zero';
+import OutputDisplay from '../OutputDisplay';
+
+const CalculatorDisplay = props => {
+  return (
+    <div className="calculator">
+    <div className="buttons">
+      <div className="numButtons">
+        <div className="zero-btn">
+          <Zero />
+        </div>
+        <div className="numRowOne">
+          <NumberButton number="1"/>
+          <NumberButton number="2" />
+          <NumberButton number="3" />
+        </div>
+        <div className="numRowTwo">
+          <NumberButton number="4" />
+          <NumberButton number="5" />
+          <NumberButton number="6" />
+        </div>
+        <div className="numRowThree">
+          <NumberButton number="7" />
+          <NumberButton number="8" />
+          <NumberButton number="9" />
+        </div>  
+        <div className="clear-btn">
+          <ClearButton />
+        </div>
+      </div>
+
+      <div className="calcAction">
+        <ActionButton action="/" />
+        <ActionButton action="x" />
+        <ActionButton action="-" />
+        <ActionButton action="+" />
+        <ActionButton action="=" />
+      </div>
+    </div>
+
+    <div className="output">
```

Comment:
Output Display should be at the top of your App since it's at the top in your design file. Not sure why it's down here. But I do see that you have it flexed so it will appear correct on the screen. Just don't know if this was intentional or not.